File: public/public-web/src/test/java/ch/difty/scipamato/publ/web/resources/IcoMoonCssResourceReferenceTest.java
package ch.difty.scipamato.publ.web.resources;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class IcoMoonCssResourceReferenceTest {
    private static final IcoMoonCssResourceReference REF = IcoMoonCssResourceReference.get();

    @Test
    void canGetInstance() {
        assertThat(REF).isInstanceOf(IcoMoonCssResourceReference.class);
    }

    @Test
    void assertResourceName() {
        assertThat(REF.getName()).isEqualTo("css/IcoMoon.css");
    }
}

File: public/public-web/src/test/kotlin/ch/difty/scipamato/publ/web/resources/IcoMoonCssResourceReferenceTest.kt
package ch.difty.scipamato.publ.web.resources

import org.amshove.kluent.shouldBeEqualTo
import org.amshove.kluent.shouldBeInstanceOf
import org.junit.jupiter.api.Test

internal class IcoMoonCssResourceReferenceTest {

    @Test
    fun canGetInstance() {
        IcoMoonCssResourceReference shouldBeInstanceOf IcoMoonCssResourceReference::class
    }

    @Test
    fun assertResourceName() {
        IcoMoonCssResourceReference.name shouldBeEqualTo "css/IcoMoon.css"
    }
}
